client/src/User.ts

```diff
@@ -0,0 +1,45 @@
+class User {
+	peerConnection: any;
+	sender: RTCRtpSender;
+	id: string;
+	stream: MediaStream;
+	player: HTMLAudioElement;
+	constructor(id: string) {
+		this.id = id;
+		this.sender = null as unknown as RTCRtpSender;
+		this.peerConnection = new RTCPeerConnection({
+			iceServers: [{ urls: "stun:iphone-stun.strato-iphone.de:3478" }],
```

Comment:
https://www.3cx.com/pbx/what-is-a-stun-server/#:~:text=A%20STUN%20(Session%20Traversal%20of,outside%20of%20the%20local%20network.&text=The%20RFC%20states%20that%20this%20port%20and%20IP%20are%20arbitrary.

Reply:
Added comment for clarification.